app/models/invenio_rdm_record_converter.rb

```diff
@@ -133,42 +135,50 @@ def creators(creators)
       # Galter Health Sciences Library is a User on production DigitalHub, but it does NOT have a formal_name value
       dh_user = User.find_by(formal_name: creator)
 
-      # organization
+      # Organization
       if organization?(creator)
-        name = creator
-        creator_type = "organisational"
-        family_name = ""
-        given_name = ""
-        identifiers = {}
-        affiliations = [] # only used for creator_type "personal"
-      # user
+        {
+          "person_or_org":
+            Hash.new.tap do |hash|
+              hash["name"] =  creator if creator.present? && !creator.include?("not identified")
+              hash["type"] = "organisational"
+            end
+        }
+      # User within DigitalHub
       elsif dh_user.present?
-        name = dh_user.formal_name # will always be this regardless of type
-        creator_type = "personal"
-        dh_user_display_name = dh_user.display_name.split(' ') # split name into components to be reused
-        family_name = dh_user_display_name.pop # remove last value from display name
-        given_name = dh_user_display_name.join(' ') # the remaining strings becomes given name
-        identifiers = dh_user.orcid.present? ? {"orcid": dh_user.orcid.split('/').pop} : {}
-        affiliations = [] # TODO: user.affiliation is nil for all users on production... where do we get this value?
-      # TODO: unknown personal, what do we enter for these fields?
+        dh_user_formal_name = dh_user.formal_name.split(',') # split name into components to be reused
+        family_name = dh_user_formal_name.shift # remove first value from formal name
+        given_name = dh_user_formal_name.join(' ') # the remaining strings becomes given name
+
+        {
+          "person_or_org":
+            Hash.new.tap do |hash|
```

Comment:
What about `name` subfield?

Reply:
<img width="763" alt="Screen Shot 2021-03-04 at 2 42 49 PM" src="https://user-images.githubusercontent.com/13144027/110027709-ee32b900-7cf7-11eb-82c5-f307cee27372.png">

Reply:
It's a little weird, but for a personal record it's not used. Organisational only.